pandas/tests/arrays/test_boolean.py

```diff
@@ -929,3 +929,23 @@ def test_diff():
     result = s.diff()
     expected = pd.Series(expected)
     tm.assert_series_equal(result, expected)
+
+
+def test_nullable_boolean_mask_series():
```

Comment:
Should these tests be removed in favor of the modified tests? May be a bit redundant with the others checking for the same / similar behavior.

Reply:
if they are truly redudant